packages/api/src/libs/queries/lsp.ts

```diff
@@ -26,13 +26,20 @@ export default (appState: Dependencies) => {
     assert(appState.stats.lsp[currency], "invalid currency: " + currency);
     return appState.stats.lsp["usd"].latest.tvl.get(address);
   }
+  function getTvm(address: string, currency: CurrencySymbol = "usd") {
+    assert(appState.stats.lsp[currency], "invalid currency: " + currency);
+    return appState.stats.lsp["usd"].latest.tvm.get(address);
+  }
   async function getFullState(state: tables.lsps.Data) {
     const collateralState = state.collateralToken ? await erc20s.get(state.collateralToken).catch(() => null) : null;
     const longTokenState = state.longToken ? await erc20s.get(state.longToken).catch(() => null) : null;
     const shortTokenState = state.shortToken ? await erc20s.get(state.shortToken).catch(() => null) : null;
     const tvl = await getTvl(state.address)
       .then((sample) => sample.value)
       .catch(() => null);
+    const tvm = await getTvm(state.address)
+      .then((sample) => sample.value)
+      .catch(() => null);
```

Comment:
Same comment here as elsewhere on burying errors. Is this something we expect in the happy path?

Reply:
Again this is mainly used to query state for the frontend. I dont want a failure in one property to prevent the rest of the state from being returned. If this is null and it shouldnt be, we can debug with logs on the ingestion side.